pkgs/ifupdown-ng.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  libbsd,
+  iproute2,
+  ...
+}:
+stdenv.mkDerivation {
+  pname = "ifupdown-ng";
+  version = "0.12.1";
```

Comment:
You're not using the tag in `src`, so by nixpkgs conventions the version is incorrect. I'd say to hell with them, but the agged `0.12.1` release is around 3 years ago, which means we should follow convention and name it something like `0-unstable-<date of commit>.

...or use the tag for `src`